src/main/kotlin/io/snyk/plugin/Utils.kt

```diff
@@ -95,6 +96,17 @@ fun getSnykToolWindowPanel(project: Project): SnykToolWindowPanel? = project.ser
 
 fun getSnykCachedResults(project: Project): SnykCachedResults? = project.serviceIfNotDisposed()
 
+fun getSnykCachedResultsForProduct(project: Project, product: ProductType): MutableMap<SnykFile, List<ScanIssue>>? {
+    return when (product) {
```

Comment:
Although IaC and Container are not retrieved from the LS, it's easier to write the code in a way that is supported for all product lines. So I've added this here